### PR TITLE
removing the 24px top margin on .modal-dialog .title

### PR DESCRIPTION
**Pull request by fentie**
### Changes
Boss HP and Difficulty on the quest details view are not aligned with their values. Removing the top margin so they line back up again, as discussed in the Aspiring Blacksmiths guild

----
UUID: 8f4a567a-038c-4c48-9941-fea2dc1d2845

cc: @Alys

### CODE REVIEW

**Comment by paglias** [comment]
Thanks! Can you post a screenshot of the changes?

**Comment by fentie** [comment]
Sure:

screen shot 2018-05-25 at 8 29 00 am

**Comment by negue** [comment]
your rule is overridden by `.modal-dialog .title ` , if you add `!important; ` then it will work on every modal

**Comment by fentie** [comment]
@negue I'm not sure I follow you. Is this a problem on other modals too? This is the only one I noticed.

**Comment by negue** [comment]
well the `margin-top` rule was overridden on the shop detail modal, also on the inventory items detail modal, so with `!important; ` it really worked everywhere 😄

**Comment by fentie** [comment]
@negue so I looked at this again and I realized part of the problem/confusion was we were using `.title` in a place where it didn't belong... it has no semantic meaning here, it's just a convenient class with a set of styles that (mostly?) match what the implementer was after.

I redid that section with a `<dl>` instead and gave it only the styles it needed.

**Comment by negue** [comment]
Could you add a `smallVersion` style, `questInfo.vue` is also used in the popovers of a quest, now it shows the big labels in that popover

File: website/client/components/shops/quests/questInfo.vue [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-
-.title {
-  text-align: left;
-  font-weight: bold;
-  white-space: nowrap;
+dt {
+  font-family: 'Roboto Condensed';
+  font-size: 20px;
+  line-height: 1.2;
+  color: #4E4A57;
```
Comment (negue) [inline website/client/components/shops/quests/questInfo.vue]: its better to use the colors from `colors.scss` , for this it would be `$gray-50`

**Comment by fentie** [comment]
@negue should be back in business now. busy few days!

File: website/client/components/shops/quests/questInfo.vue [hunk 2]
```diff
@@ -49,6 +51,17 @@
 }
 </style>
 
+<style lang="scss">
+.questPopover {
+  dt {
+    font-family: 'Roboto', sans-serif;
```
Comment (paglias) [inline website/client/components/shops/quests/questInfo.vue]: this font-family rule can be removed as it's specified at an higher level

**Review by paglias — CHANGES_REQUESTED** [review]

File: website/client/components/shops/quests/questInfo.vue [hunk 3]
```diff
@@ -21,10 +22,11 @@
 <style lang="scss" scoped>
 @import '~client/assets/scss/colors.scss';
 
-.title {
-  text-align: left;
-  font-weight: bold;
-  white-space: nowrap;
+dt {
+  font-family: 'Roboto Condensed';
```
Comment (paglias) [inline website/client/components/shops/quests/questInfo.vue]: same here

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by fentie** [comment]
@paglias updated, rebased, and pushed

**Comment by paglias** [comment]
this is looking good @fentie, thanks! We're going to merge it at the end of this week or at the beginning of the next one

**Comment by paglias** [comment]
thanks @fentie ! notated towards your next tier

**Pull request merged**